account_invoice_import/wizard/account_invoice_import.py

```diff
@@ -1005,8 +1022,7 @@ def message_new(self, msg_dict, custom_values=None):
                 logger.info(
                     'Attachment %d: %s. Trying to import it as an invoice',
                     i, attach.fname)
-                parsed_inv = self.parse_invoice(
-                    base64.b64encode(attach.content), attach.fname)
```

Comment:
this change doesn't look LG, here work is happening on multiple attachments relying on file on wizard is wrong